eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -96,16 +100,20 @@ library
         transformers,
         validation-selective
 
+    if flag(asserts)
+        ghc-options: -fno-ignore-asserts
+
 library testlib
     exposed-modules:
         Test.Cardano.Ledger.Conway.Arbitrary
         Test.Cardano.Ledger.Conway.Binary.Cddl
         Test.Cardano.Ledger.Conway.Binary.RoundTrip
         Test.Cardano.Ledger.Conway.ImpTest
         Test.Cardano.Ledger.Conway.Imp
+        Test.Cardano.Ledger.Conway.Imp.GovSpec
```

Comment:
I wonder why you moved this here, it seems to make more alphabetical sense after `EnactState` where it was, no?

Reply:
Yes agree :100: . The previous position has been restored. :raised_hands: